Instruments/include/TTFileDescriptor.h

```diff
@@ -0,0 +1,106 @@
+/*! Definition of the file descriptor for TTbar sample merging.
+This file is part of https://github.com/hh-italian-group/hh-bbtautau. */
+
+#pragma once
+
+#include <list>
+#include <istream>
+#include <ostream>
+#include "AnalysisTools/Core/include/EnumNameMap.h"
+#include "AnalysisTools/Core/include/NumericPrimitives.h"
+#include "AnalysisTools/Core/include/PhysicalValue.h"
+#include "h-tautau/Analysis/include/EventTuple.h"
+#include "h-tautau/Analysis/include/AnalysisTypes.h"
+#include "AnalysisTools/Core/include/ConfigReader.h"
+#include "Instruments/include/SampleDescriptor.h"
+
+namespace analysis {
+
+namespace sample_merging{
+
+struct TTBinDescriptor {
+    std::string name;
+    std::vector<std::string> file_paths;
+    FileType fileType;
+    Range<int> genType;
+    std::string channel;
+
+    PhysicalValue nu;
+    PhysicalValue weight;
+    size_t inclusive_integral;
+
+    TTBinDescriptor()
+        : genType(0,0),nu(0.0, std::numeric_limits<double>::infinity()),
+          weight(std::numeric_limits<double>::quiet_NaN()) {}
+
+    bool Contains(const analysis::GenEventType& genEventType) const
+    {
+        return genType.Contains(static_cast<int>(genEventType));
+    }
+
+    static std::vector<TTBinDescriptor> LoadConfig(const std::string& config_file)
+    {
+        std::vector<TTBinDescriptor> ttBinDescriptors;
+        ttBinDescriptors.clear();
+        std::ifstream cfg(config_file);
+        while (cfg.good()) {
+            std::string cfgLine;
+            std::getline(cfg,cfgLine);
+            if (!cfgLine.size() || cfgLine.at(0) == '#') continue;
+            auto columns = ConfigEntryReader::ParseOrderedParameterList(cfgLine, true);
+            std::istringstream ss(cfgLine);
+            TTBinDescriptor descriptor;
+            if(columns.size() >= 2)
+                ss >> descriptor.genType;
+            if (columns.size() >= 7){
+                double col_weight = analysis::Parse<double>(columns.at(2));
+                double col_weight_err = analysis::Parse<double>(columns.at(3))*col_weight;
+                descriptor.weight = PhysicalValue(col_weight,col_weight_err);
+                double col_nu = analysis::Parse<double>(columns.at(4));
+                double col_nu_err = analysis::Parse<double>(columns.at(5))*col_nu;
+                descriptor.nu = PhysicalValue(col_nu,col_nu_err);
+            }
```

Comment:
inclusive_integral not parsed. Please test LoadConfig method by passing output config as an input. (same for DY)